resources/public/api/conf/1.0/reliefOther_retrieve.raml

```diff
@@ -0,0 +1,27 @@
+is:
+  - headers.acceptJson
+  - headers.contentTypeJson
```

Comment:
I remember you saying about the content Type on a retrieve & delete

Bit unsure if it should be here.